src/main/resources/htmlpublisher/HtmlPublisherTarget/config.jelly

```diff
@@ -10,6 +10,11 @@
     <f:textbox default="index.html"/>
   </f:entry>
 
+    <f:entry field="reportTitles" title="${%reportTitles.title}">
+      <f:textbox default="index"/>
```

Comment:
Iif I had an old job before this existed, then I click configure, don't change anything, and save, it gets this "index" title as the value. I also think that the default of using the filename is better than "index". Can it just be empty?

Also, this field has no help text like the others do. It should explain if they are comma separated, and where they will be visible. Thanks!